Tests/ML/augmentations/test_transform_pipeline.py

```diff
@@ -7,45 +7,64 @@
 import PIL
 import pytest
 import torch
-from torchvision.transforms import CenterCrop, ColorJitter, RandomAffine, RandomErasing, RandomHorizontalFlip, \
-    RandomResizedCrop, Resize, ToTensor
+from torchvision.transforms import (
+    CenterCrop,
+    ColorJitter,
+    RandomAffine,
+    RandomErasing,
+    RandomHorizontalFlip,
+    RandomResizedCrop,
+    Resize,
+    ToTensor,
+)
```

Comment:
The formatting changes to this file are produced by the black option. 
I find an automatic formatting standard helps and Black is very popular, but I am not particularly attached to any standard. If this is something already discussed and avoided on purpose I can eliminate the formatting from the list of pre-commit hooks and revert this file